inception-project/inception
# #2250 - Include user preferences in the project export

**What's in the PR**
- Added exporter for user preferences
- Added test
- Move around code to avoid circular dependency when the user preferences importer requires user permissions to be imported first

**How to test manually**
* Set a project-specific preference, e.g. open a sidebar on the annotation page
* Export project
* Import back
* Sidebar should still be open when accessing the annotation page in the new project

**Automatic testing**

**Documentation**
